src/main/java/com/glodblock/github/loader/ChannelLoader.java

```diff
@@ -78,6 +79,10 @@ public void run() {
                 CPacketLevelTerminalCommands.class,
                 id++,
                 Side.SERVER);
+        netHandler
+                .registerMessage(new CPacketPatternMultiSet.Handler(), CPacketPatternMultiSet.class, id++, Side.SERVER);
+        netHandler
+                .registerMessage(new CPacketPatternMultiSet.Handler(), CPacketPatternMultiSet.class, id++, Side.CLIENT);
```

Comment:
This seems to be a S->C only packet so registering for both sides shouldn't be needed

Reply:
real, done